lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -245,6 +259,26 @@ class MetroItinerary<Props> extends NarrativeItinerary {
       currency
     )
 
+    const co2VsBaseline = Math.round(itinerary.co2VsBaseline * 100)
+    const emissionsNote = !mini && co2VsBaseline < -20 && (
+      <>
+        <ItineraryNoteIcon type="leaf" />
+        <FormattedNumber
+          style="unit"
+          unit="percent"
+          unitDisplay="narrow"
+          value={Math.abs(co2VsBaseline)}
+        />{' '}
```

Comment:
I don't like the `{' '}`. Would it be possible to insert the number as a value inside the message and then pass this entire component in as the value?